src/components/modals/data-table-modal.js

```diff
@@ -30,183 +32,179 @@ import {Clock} from 'components/common/icons/index';
 // Breakpoints
 import {media} from 'styles/media-breakpoints';
 
-const ReactDataGrid = window.navigator ? require('react-data-grid/dist/react-data-grid.min') : null;
-
-let shouldPreventScrollBack = false;
-
-if (window.navigator && window.navigator.userAgent) {
```

Comment:
This is to prevent scrolling left and right in the data table causes browser goes back to previous page. It would be extremely frustrating for someone to accidentally go back to the previous page and lose all his/her kepler.gl map. Please add this check back on `onMouseWheel`

![ezgif com-optimize (14)](https://user-images.githubusercontent.com/3605556/59907264-88858880-93bf-11e9-9064-6ddeaab5af12.gif)